patterns/1-initial/operational-model.md

```diff
@@ -0,0 +1,68 @@
+## Title
+
+InnerSource Operational Model
+
+## Patlet
+
+InnerSource is key across all of the software production chain. This includes from idea to delivery to customer. This pattern defines the software development life cycle from an InnerSource perspective.
+
+## Problem
+
+InnerSource is typically focused on letting developers work together. There are several patterns focused on specific roles as the Trusted Committer or from a more organization level as the Review Committee. However, it is still not clear how all of this works together.
+
+This pattern aims at providing an overview to management and Chief level of how the software life cycle may be in a more InnerSource way with the goal of having a place where InnerSource principles may apply to the several people involved: from Product Owners to Contributors.
```

Comment:
maybe: ...how the software life cycle may be **managed** in a more InnerSource way..